src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1449,6 +1449,90 @@ void valid() {
                 assertAndDump(model, "NIST.valid: confidence must be between 0 and 1", nistSeedlingValidator, true);
             }
         }
+
+        // CompoundJustification must be used only for justifications of argument assertions,
+        // and not for justifications for entities, events, or relation KEs
+        @Nested
+        class RestrictCompoundJustification {
```

Comment:
It might be a good idea to combine this test with EdgeJustificationCompound. Then we don't have two tests (EdgeJustificationCompound.valid and RestricCompoundJustification.valid) doing the same thing.

Reply:
Merged EdgeJustificationCompound and RestrictCompoundJustification into single test called RestrictJustification.